dataservice/api/sequencing_experiment/schemas.py

```diff
@@ -3,15 +3,32 @@
 from dataservice.api.sequencing_experiment.models import SequencingExperiment
 from dataservice.api.common.schemas import BaseSchema
 from dataservice.api.common.custom_fields import DateOrDatetime
-from dataservice.api.common.validation import validate_positive_number
+from dataservice.api.common.validation import (validate_positive_number,
+                                               enum_validation_generator)
 from dataservice.extensions import ma
 
+EXPERIMENT_STRATEGY_ENUM = {'WGS', 'WXS', 'RNA-Seq', 'Other'}
```

Comment:
I just had a discussion with @allisonheath and `miRNA-Seq` needs to be added since it should be distinguished from `RNA-Seq`.

Reply:
👍